vllm/config.py

```diff
@@ -3093,16 +3100,7 @@ def model_post_init(self, __context: Any) -> None:
         assert count_none + count_all <= 1, "Can only specify 'none' or 'all'"
 
         if self.splitting_ops is None:
-            if envs.VLLM_USE_V1:
```

Comment:
@SageMoore @ProExpertProg @bnellnm - do you guys think that the way I did this is okay?

Reply:
This seems fine although I think we should default the `splitting_ops` field to an empty list, not to none, so you don't need this extra checking.

Reply:
Also, why not just do
```python
if self.use_v1:
    self.splitting_ops = [...]
```

Reply:
Does it have to get set later? If yes could you add a comment why it happens later?

Reply:
yeah, we need to have the `ModelConfig` before we decide if V0 or v1